### PR TITLE
docs: update edge case for ConfigModule

### PR DESCRIPTION
**Pull request by PhearZero**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/docs.nestjs.com/blob/master/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Docs
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

**Review by micalevisk — APPROVED** [review]

File: content/techniques/configuration.md [hunk 1]
```diff
@@ -565,3 +565,29 @@ You can then use it as usual, by calling the `get` method with the configuration
...
+If a decorator depends on an environment variable's existance, `dotenv` can be used directly in the `main.ts` file. This is particularly useful for the `@Cron` decorator.
+
+Preload the environment variables:
+
+```typescript
+@@filename(main)
+import dotenv from 'dotenv'
+dotenv.config()
+import {AppModule} from ./app.module
+```
```
Comment (jmcdo29) [inline content/techniques/configuration.md:580]: Can we make some sort of mention here that the import and `config()` call **have** to happen before importing `./app.module`? If it isn't, then `process.env` will not be defined at the time of the decorator function's execution

**Review by jmcdo29 — APPROVED** [review]

**Review by Tony133 — APPROVED** [review]

**Comment by kamilmysliwiec** [comment]
I wonder if we should use `--env-file` flag in this case instead

**Comment by micalevisk** [comment]
yeah, having to import `dotenv` ourselves seems odd.

**Comment by kamilmysliwiec** [comment]
To make things simpler, we might expose the `--env-file` flag on the `nest start` command and just pass it down to `node` (instead of forcing everyone to use --exec etc)

**Comment by PhearZero** [comment]
It's only a few more months before Node 18 hits end of life, feel free to close the ticket in favor of a `--env-file` pass-through

**Comment by kamilmysliwiec** [comment]
https://github.com/nestjs/nest-cli/pull/2818

**Pull request closed without merging**